.github/workflows/build_release_artifacts.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Release artifacts
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  trigger_azdo_ci:
+    name: Download and Build artifacts
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/download-artifact@v3
+      with:
+        path: ${{ runner.temp }}\grpc-labview-artifacts
+
+    - name: Stage Artifacts
+      run:
+       python ${{runner.workspace}}\grpc-labview\build-it\stage_artifacts.py --downloaded_path ${{ runner.temp }}\grpc-labview-artifacts --staging_path ${{ runner.temp }}\grpc-labview-staged-artifacts

```

Comment:
can we use the step output from download-artifact and pass it on to the python script here as --downloaded_path